klippy/extras/bltouch.py

```diff
@@ -41,6 +43,10 @@ def __init__(self, config):
         mcu = pin_params['chip']
         mcu.register_config_callback(self._build_config)
         self.mcu_endstop = mcu.setup_pin('endstop', pin_params)
+        # output mode
+        self.output_mode = config.getchoice('set_output_mode',
+                                            {'5V': '5V', '0V': '0V',
+                                             None: None}, None)
```

Comment:
Sometimes the code uses "OD" and sometimes "0V" - that doesn't look correct.

Also, None is not the same as "None" - the way the config reading here is setup, the only way to select None is to not specify a set_output_value option in the config file.  I, personally, think that is preferable and that the docs should be updated to say something like "If set_output_mode is specified then the given mode is set at startup.  The default is to not set a mode at startup".

Reply:
0V vs. OD, oof, copy and paste is evil. OD is for "OPEN DRAIN". 0V would be factually wrong. And I'll update the docs.